addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1305,13 +1307,43 @@ export class OdooEditor extends EventTarget {
     historyResetFromSteps(steps, historyIds) {
         this._historyIds = historyIds;
         this.observerUnactive();
+
+        // Recompute `_latestComputedSelection` after node recreation by converting `anchorNode`
+        // and `focusNode` to paths relative to the editable container for later lookup.
+        let latestComputedSelectionPath = null;
```

Comment:
No need to forcefully initialize the variable in js. All unitialized variables have a default `undefined` value.